packages/search-engine/src/search-engine.js

```diff
@@ -0,0 +1,39 @@
+const fastify = require('fastify');
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+
+function searchEngine() {
+  // TODO
+}
+
+dotenv.config();
+// initialized Fastify App
+const app = fastify();
+
+const mongoURI = process.env.CONNECT_DB;
+
+// connected fastify to mongoose
+try {
+  mongoose.connect(mongoURI);
+  console.log('DB Connected...');
+} catch (e) {
+  console.error(e);
+}
+
+app.get('/', (request, reply) => {
+  try {
+    reply.send('Welcome to the search');
+  } catch (e) {
+    console.error(e);
+  }
+});
+
+app.listen(5000, (err, address) => {
+  if (err) {
+    console.error(err);
+    process.exit(1);
+  }
+  console.log(`Server running on ${address}`);
+});
```

Comment:
@JackoDev why are we creating an additional server? the idea is to integrate the usecases of this package with the api-gw, so with that on mind we wouldn't need to configure an additional server